docs/element-definitions.md

```diff
@@ -123,10 +123,10 @@ The Resource representation with its available transitions and its data.
 #### Properties
 
 - `element`: resource (string, fixed)
-- `attributes` (object)
+- `attributes`
     - `href` (Templated Href) - URI Template of this resource.
     - `hrefVariables` (Href Variables) - Definition of URI Template variables used in the `href` property.
-- `content` (array)
+- `content` (array, fixed-type)
```

Comment:
I am not sure about `fixed-type` either. It means that I can expect only predefined types here so new types can't go in without being it a breaking change, do we want this?